include/swift/SIL/SILInstruction.h

```diff
@@ -7424,6 +7441,31 @@ class MoveValueInst
   void setAllowsDiagnostics(bool newValue) { allowDiagnostics = newValue; }
 };
 
+/// Equivalent to a copy_addr [take] to [initialization] except that it is used
```

Comment:
This is an incorrect comment. I am going to fix it. This is equivalent to a copy_addr [init].